daffodil-core/src/main/scala/org/apache/daffodil/grammar/Grammar.scala

```diff
@@ -27,14 +27,14 @@ import org.apache.daffodil.compiler.ForParser
 import org.apache.daffodil.processors.unparsers.NadaUnparser
 import org.apache.daffodil.processors.parsers.NadaParser
 
-abstract class UnaryGram(context: Term, rr: => Gram) extends NamedGram(context) {
-  private lazy val r = rr
-
-  final override lazy val gram = {
-    if (r.isEmpty) EmptyGram
-    else this
-  }
-}
+//abstract class UnaryGram(context: Term, rr: => Gram) extends NamedGram(context) {
```

Comment:
Delete comment.

Reply:
If this is your only comment I need an approval so I can fix this and push it.

Reply:
I think usually Apache just consideres a +1 comment  approval and doesn't require the actual GitHub "Aprove" thing, especially since many communities dont' even use github. But as a communiy we can decide that we can require the github "approve" thing as well. Might be worth starting a conversastion on dev@ if you'd prefer that. I have no objectiosn to it, and there are some benefits. I'll also marked this as approved.